src/app/drive/components/DriveExplorer/DriveExplorerGrid/DriveExplorerGrid.tsx

```diff
@@ -30,34 +33,51 @@ class DriveExplorerGrid extends React.Component<DriveExplorerGridProps> {
   }
 
   get itemsFileList(): JSX.Element[] {
-    return this.props.items.filter((item) => !item.isFolder).map((item) => {
-      const itemParentId = item.parentId || item.folderId;
-      const itemKey = `'file'-${item.id}-${itemParentId}`;
+    return this.props.items
+      .filter((item) => !item.isFolder)
+      .map((item) => {
+        const itemParentId = item.parentId || item.folderId;
+        const itemKey = `'file'-${item.id}-${itemParentId}`;
 
-      return <DriveExplorerGridItem key={itemKey} item={item} />;
-    });
+        return <DriveExplorerGridItem key={itemKey} item={item} />;
+      });
   }
 
   get itemsFolderList(): JSX.Element[] {
-    return this.props.items.filter((item) => item.isFolder).map((item) => {
-      const itemParentId = item.parentId || item.folderId;
-      const itemKey = `'folder'-${item.id}-${itemParentId}`;
+    return this.props.items
+      .filter((item) => item.isFolder)
+      .map((item) => {
+        const itemParentId = item.parentId || item.folderId;
+        const itemKey = `'folder'-${item.id}-${itemParentId}`;
 
-      return <DriveExplorerGridItem key={itemKey} item={item} />;
-    });
+        return <DriveExplorerGridItem key={itemKey} item={item} />;
+      });
   }
 
   render(): ReactNode {
-    const { isLoading } = this.props;
+    const { isLoading, onEndOfScroll, hasMoreItems } = this.props;
 
-    return <div className="files-grid flex-grow">{isLoading ? (
-      this.loadingSkeleton
-    ) : (
+    return (
       <>
-        { this.itemsFolderList}
-        { this.itemsFileList}
+        {isLoading ? (
+          <div className="files-grid flex-grow">this.loadingSkeleton</div>
```

Comment:
this.loadingSkeleton should be in curly braces for it to be evaluated

Reply:
corrected!